docker/mint/docker-run.sh

```diff
@@ -1,18 +1,22 @@
-#!/bin/sh
+#!/bin/bash
 # Copyright (c) The Libra Core Contributors
 # SPDX-License-Identifier: Apache-2.0
 
 set -ex
 export RUST_BACKTRACE=full
 
-seed=""
+declare -a params
 if [ -n "${CFG_SEED}" ]; then
-    seed="-s $CFG_SEED"
+    params+="-s $CFG_SEED "
+fi
+
+if [ -n "${CFG_NUM_VALIDATORS}" ]; then # Total number of nodes in this network
+	  params+="-n ${CFG_NUM_VALIDATORS} "
 fi
 
 /opt/libra/bin/config-builder faucet \
     -o /opt/libra/etc \
-    $seed
+    ${params[@]}
```

Comment:
forgive my naïveté what happens if there are no parameters specified?

Reply:
I have no idea what is this, copied from other script